projects/angular/src/layout/API.md

```diff
@@ -0,0 +1,59 @@
+# Card - Style API
+
+## CSS Variables
+
+| CSS Variable Name                      | Description                                                    |
+| -------------------------------------- | -------------------------------------------------------------- |
+| --clr-card-bg-color                    | Changes the background color of card.                          |
+| --clr-card-divider-color               | Changes the divider color of the card.                         |
+| --clr-card-title-color                 | Changes the text color of card header.                         |
+| --clr-card-title-font-weight           | Changes the thickness of card title.                           |
+| --clr-card-border-width                | Changes the border width of the card.                          |
+| --clr-card-border-radius               | Changes the border radius of card.                             |
+| --clr-card-border-color                | Changes the border color of the card.                          |
+| --clr-card-box-shadow-color            | Changes the background shadow color of card.                   |
+| --clr-card-clickable-border-color      | Changes the background color of clickable card.                |
+| --clr-card-clickable-box-shadow-color  | Changes the background shadow color of clickable card.         |
+
+
+## Class names
+
+| Class name            | Description                                                    |
+| --------------------- | -------------------------------------------------------------- |
+| card                  |  Card is a content container for text, photos, and actions in the context of a single subject. |
+| card-title            |  A title within the header.                                                                    |
```

Comment:
The `.card-title` does not go in the header. It goes inside a `.card-block`. I am going to fix this myself.

Reply:
Okay, I appreciate you informing me.